jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/query/model/QueryDefinition.java

```diff
@@ -33,7 +33,8 @@
         FILTERED_PROCESS,
         FILTERED_BA_TASK,
         FILTERED_PO_TASK,
-        CUSTOM;
+        CUSTOM,
+        USER_GROUPS_TASK;
```

Comment:
I would leave CUSTOM as the last one, by looking at the former PRs this was the convention before.

Reply:
Done

Reply:
Thanks, Maria.

Reply:
My pleasure :-)